### ISSUE TITLE
getplastertemplate.ps1 from Prior VSCode Install Is Still Executing When Open Default PowerShell Console

### ISSUE THREAD

**Issue by HJLBX** [PowerShell/vscode-powershell#1249]
Uninstalled VSCode but now every time I launch the default PowerShell, getplastertemplate.ps1 keeps executing and it is triggering the installed security software - which blocks it and creates alert notifications:

03/26/18 04:12:04 Prevented process <getplastertemplate.ps1 | C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe> from launching from <c:\users<user>.vscode\extensions\ms-vscode.powershell-1.6.0\modules\plaster>.

How do I stop the getplastertemplate.ps1 from exectuing every time I open the default PowerShell now ?

Visual Studio Code has been completely uninstalled and a manual cleanup of both the file system and the registry have been performed.

In other words, c:\users<user>.vscode\extensions\ms-vscode.powershell-1.6.0\modules\plaster does not exist.

Also, all the powershell profiles are empty.

How do I reset PowerShell to the way it was before installing VSCode ?

**Comment by HJLBX** [comment]
There is a file named ModuleAnalysisCache that was created when VSCode was installed in 

C:\Users\<users>\AppData\Local\Microsoft\Windows\PowerShell

When ModuleAnalysisCache is renamed (or deleted), the issue stops.

**Comment by tylerl0706** [comment]
Thanks for the report!

Can you share with us what is in your PSModulePath?

**Comment by HJLBX** [comment]
C:\Users\Lockdown\Documents\WindowsPowerShell\Modules;

C:\Program Files\WindowsPowerShell\Modules;

C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules

**Comment by tylerl0706** [comment]
ok that looks normal... How about:

```
$env:PSDisableModuleAnalysisCacheCleanup
```
Is that set to 1 or 0?

**Comment by tylerl0706** [comment]
Also, do you have Plaster in your PSModulePath?

**Comment by HJLBX** [comment]
$env:PSDisableModuleAnalysisCacheCleanup is not returning any value.

Plaster is not in my PSModulePath.

**Comment by tylerl0706** [comment]
Very strange. Thanks for the report.

**Comment by SydneyhSmith** [comment]
@HJLBX what is in your PowerShell profile, is there anything in there that could be causing this?

**Comment by HJLBX** [comment]
The PowerShell profile at that time was the default.

Since this issue is over 1 year old, the system state and the exact details of what happened at that time are long gone.

**Comment by SydneyhSmith** [comment]
@HJLBX thanks for the response, I see now that you had indicated that the profile was empty in the initial issue. I am going to close for now since we are unable to reproduce, but will re-open if this happens again. Thanks for opening the issue, and following up!
